src/game/mod.rs

```diff
@@ -9,10 +9,10 @@
 use music;
 use opengl_graphics::GlGraphics;
 use opengl_graphics::glyph_cache::GlyphCache;
-use piston_window::{Button, clear, Context, Input, Key, PistonWindow, Size, text, Transformed,
```

Comment:
Yeah they have been planning to alphabetize imports for a while.  We used to do this manually, but now the tool does it for it, which is nice.